test/test_deploy.py

```diff
@@ -37,43 +40,28 @@ def get_deploy_transaction(inputs: List[int], salt=0):
     )
 
 
+@pytest.mark.parametrize(
+    "dev_net_args, expected_tx_hash, expected_block_hash",
+    [
+        (
+            [*PREDEPLOY_ACCOUNT_CLI_ARGS],
+            "0x615badf1d4446082f598fa16416d4d3623dfb8cc5d58276515f502f8fa22009",
```

Comment:
To respond to the comment I was tagged in, the hash is not the same as in the general workflow test because there it is deployed with salt=0x99, here the default 0 is being used